demo/runners/agent_container.py

```diff
@@ -1065,32 +1067,46 @@ async def test_main(
             seed=None,
             aip=aip,
         )
+        airops_container = AgentContainer(
+            genesis,
+            "Airops.agent",
+            start_port,
+            no_auto=no_auto,
+            revocation=revocation,
+            tails_server_base_url=tails_server_base_url,
+            show_timing=show_timing,
+            multitenant=multitenant,
+            mediation=mediation,
+            use_did_exchange=use_did_exchange,
+            wallet_type=wallet_type,
+            public_did=True,
+            seed="random",
+            cred_type=cred_type,
+            aip=aip,
+        )
+        caa_container = AgentContainer(
+            genesis,
+            "caa.agent",
+            start_port,
+            no_auto=no_auto,
+            revocation=revocation,
+            tails_server_base_url=tails_server_base_url,
+            show_timing=show_timing,
+            multitenant=multitenant,
+            mediation=mediation,
+            use_did_exchange=use_did_exchange,
+            wallet_type=wallet_type,
+            public_did=True,
+            seed="random",
+            cred_type=cred_type,
+            aip=aip,
+        )
 
         # start the agents - consortiq gets a public DID and schema/cred def
-        await consortiq_container.initialize(
-            schema_name="degree schema",
-            schema_attrs=[
-                "name",
-                "date",
-                "degree",
-                "grade",
-            ],
-        )
+        await consortiq_container.initialize()
```

Comment:
should we update agent names (LABELS) across the board e.g.
licensee,
autority,
...
rather than airops, caa etc?

@mattdean-digicatapult @jonmattgray